server/index.js

```diff
@@ -26,44 +26,56 @@ app.get('/:product_id', (req, res) => {
 app.get('/system_req/:product_id', (req, res) => {
   console.log('got it');
   const id = req.params.product_id;
-  Overview.find({ product_id: id }).then((doc) => {
-    axios
-      .get(`http://ec2-54-224-38-115.compute-1.amazonaws.com:5150/genre/${id}`)
-      .then((response) => {
-        const resArray = doc;
-        console.log('resArray: ', resArray);
-        const newGenre = response.data;
-        console.log('newGenre: ', newGenre);
-        const steamNumber = resArray[0].steam_rating;
-
-        resArray.push(newGenre);
-
-        if (steamNumber) {
-          const describeSteamRating =
-            steamNumber >= 95
-              ? 'Overwhelmingly Positive'
-              : steamNumber >= 80
-                ? 'Very Postive'
-                : steamNumber >= 70
-                  ? 'Mostly Positive'
-                  : steamNumber >= 40
-                    ? 'Mixed'
-                    : steamNumber >= 20
-                      ? 'Mostly Negative'
-                      : 'Very Negative';
-          resArray.push(describeSteamRating);
-        }
-
-        return resArray;
-      })
-      .then((resArray) => {
-        res.set({ 'Access-Control-Allow-Origin': '*' });
-        res.send(resArray);
-      })
-      .catch((error) => {
-        throw error;
-      });
-  });
+  console.log('id: ', id);
```

Comment:
server/index.js: Overall the endpoints and their containing functions are organized well, and make sense. Again, I would argue for increased modularity, by moving the database-interfacing functions to their own files, then importing them and using them here. It's better for code clarity & being able to edit each interface completely separate of the other(s).